conan/tools/qbs/qbs.py

```diff
@@ -55,6 +55,9 @@ def resolve(self, parallel=True):
         if self.profile:
             args.append('profile:%s' % self.profile)
 
+        if os.path.exists(os.path.join(self._conanfile.source_folder, 'conan-qbs-deps')):
```

Comment:
There might be a couple of issues here:
- It is not expected in general that the ``conan-qbs-deps`` folder is in the ``source_folder``, because it is typically temporary files not intended to be committed to source control
- Typically generated files should be generated in the ``generators_folder`` and read from there by helpers such as ``Qbs``